src/templating/filesystem.ts

```diff
@@ -162,7 +162,7 @@ export async function writeFiles(
 
   const newKeys = context.env.newEnvironmentVariableKeys;
   if (newKeys.length > 0) {
-    console.log(
+    logger.info(
```

Comment:
Should this use `writePlainOutput` instead of logging as it is a message to the user?

Reply:
No because in that case you want to write to `stderr` and keep `stdout` empty. Here's a good example. https://www.jstorimer.com/blogs/workingwithcode/7766119-when-to-use-stderr-instead-of-stdout

Reply:
Ok, good article! :+1: I'm convinced.